web/blog/2023-04-17-How-I-Built-CoverLetterGPT.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'How I Built CoverLetterGPT - SaaS app with the PERN stack, GPT, Stripe, & Chakra UI'
+authors: [vinny]
+tags: [wasp, ai, gpt, fullstack, PERN, stripe, chakra, saas]
+---
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import InBlogCta from './components/InBlogCta';
+import WaspIntro from './_wasp-intro.md';
+import ImgWithCaption from './components/ImgWithCaption'
+
+<br/>
+<div style={{ textAlign: "center", width: "100%", height: "400px", display: "inline-block" }}>
+<iframe height="100%" width ="100%" src="https://www.youtube.com/embed/D1l0iwGUed0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="true"></iframe>
+</div>
+
+<!--truncate-->
+
+---
+
+Like many other software developers, I really enjoy trying out new technologies even if it's just to get a feel for what they can do. 
+
+So when I first learned about the [OpenAI API](https://platform.openai.com/docs/api-reference), I knew I wanted to give it a try. I had already wanted to create a SaaS app that could help manage the process of applying to numerous jobs, and the prospect of adding GPT into the mix made it even more interesting. So with API access and a bit of free time, I decided to give it a shot.
+
+I threw together a simple version of the app in about 3-4 days and [CoverLetterGPT](https://coverlettergpt.xyz) was born, a SaaS app that uses GPT-3.5-turbo to generate, revise, and manage cover letters for you based on your skills and the specific job descriptions. 
+
+Even though I did think it had potential as a SaaS app, I was approaching it mostly as way to learn how to build one for the first time. And after seeing so many people "building in public" and sharing their progress, I thought it would be fun to try it out myself.
```

Comment:
```suggestion
Even though I did think it had potential as a SaaS app, I was approaching it mostly as a way to learn how to build one for the first time. And after seeing so many people "building in public" and sharing their progress, I thought it would be fun to try it out myself.
```
Grammarly